### PR TITLE
[screengrab] AAR version bump

### PR DESCRIPTION
**Pull request by mfurtak**
* Prevent users on Android N from attempting to use internal storage (#7838)

### CODE REVIEW

**Review by asfalcone — APPROVED** [review]

File: screengrab/version.properties [hunk 1]
```diff
@@ -1,3 +1,3 @@
 major=1
 minor=0
-patch=0
+patch=1
```
Comment (mfurtak) [inline screengrab/version.properties]: test comment

Comment (mfurtak) [inline screengrab/version.properties]: test comment 2

Comment (mfurtak) [inline screengrab/version.properties]: test comment 3

Comment (hjanuschka) [inline screengrab/version.properties]: comment works 👍

Comment (mfurtak) [inline screengrab/version.properties]: Hm, sorry about the email! 😅

**Pull request merged**